docs/source/workflows/design_spaces.rst

```diff
@@ -128,3 +129,34 @@ As an example, an enumerated design space that represents points from a 2D Carte
    )
 
    registered_design_space = project.design_spaces.register(design_space)
+
+.. _data-source-design-space:
```

Comment:
I don't think you need this tag unless you're referencing this section from another page.